payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/CardNumberController.kt

```diff
@@ -71,6 +73,7 @@ internal class DefaultCardNumberController(
     override val initialValue: String?,
     override val showOptionalLabel: Boolean = false,
     private val cardBrandChoiceConfig: CardBrandChoiceConfig = CardBrandChoiceConfig.Ineligible,
+    private val cardBrandFilter: CardBrandFilter = DefaultCardBrandFilter,
```

Comment:
Before we move this to private beta (not this PR), we will want and end to end/integration test ensuring this gets propagated all the way from the configuration.